diesel/src/macros/ops.rs

```diff
@@ -78,6 +80,7 @@ macro_rules! date_time_expr {
 #[doc(hidden)]
 macro_rules! __diesel_generate_ops_impls_if_date_time {
     ($column_name:ident, Nullable<$($inner:tt)::*>) => { __diesel_generate_ops_impls_if_date_time!($column_name, $($inner)::*); };
+    ($column_name:ident, Unsigned<$($inner:tt)::*>) => { __diesel_generate_ops_impls_if_date_time!($column_name, $($inner)::*); };
```

Comment:
There are no unsigned date/time types in diesel, therefore we should not add this here.
```suggestion
```